scripts/ai-sdlc/generate-impl.mjs

```
@@ -0,0 +1,217 @@
+}
+
+if (!parsed || !Array.isArray(parsed.files) || parsed.files.length === 0) {
+  console.error('No files in response:', JSON.stringify(parsed, null, 2));
+  process.exit(1);
+}
+
+// Write files
+const writtenPaths = [];
+const repoRoot = process.cwd();
+for (const { path, content } of parsed.files) {
+  if (!path || !content) continue;
```

ESLint

[error] Expected { after 'if' condition.

(curly)